G7SensorKitUI/Views/G7StartupView.swift

```diff
@@ -26,7 +26,7 @@ struct G7StartupView: View {
                     .frame(height: 120)
                     .padding(.horizontal)
             }.frame(maxWidth: .infinity)
-            Text(LocalizedString("Loop can read G7 CGM data, but you must still use the Dexcom G7 App for pairing, calibration, and other sensor management.", comment: "Descriptive text on G7StartupView"))
```

Comment:
What is the reason for this text change?

Reply:
The idea is that if you use a generic term for the app, a different open source app can use the same submodule and the translations would not require an update from the appName. 

Specifically, the OpenAPS implementation on the iPhone platform intends to use the same versions as are found in the LoopKit repositories.

Reply:
@ps2 Let me know if you want me to revert the language change.

Reply:
I only touched the hard coded “Loop”. Do you want me to add appName instead?

Reply:
Modified to use appName in text string